diff --git a/node_modules/expo-file-system/ios/EXFileSystem.podspec b/node_modules/expo-file-system/ios/EXFileSystem.podspec
index d47ce72..3e14a03 100644
--- a/node_modules/expo-file-system/ios/EXFileSystem.podspec
+++ b/node_modules/expo-file-system/ios/EXFileSystem.podspec
@@ -10,16 +10,11 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '11.0'
+  s.platforms     = {:ios => '11.0', :tvos => '12.0'}
   s.source         = { :git => 'https://github.com/expo/expo.git' }
 
   s.dependency 'UMCore'
   s.dependency 'UMFileSystemInterface'
 
-  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
-    s.source_files = "#{s.name}/**/*.h"
-    s.vendored_frameworks = "#{s.name}.xcframework"
-  else
-    s.source_files = "#{s.name}/**/*.{h,m}"
-  end
+  s.source_files = "#{s.name}/**/*.{h,m}"
 end
