diff --git a/node_modules/expo-font/ios/EXFont.podspec b/node_modules/expo-font/ios/EXFont.podspec
index 9b88efb..a2ecd59 100644
--- a/node_modules/expo-font/ios/EXFont.podspec
+++ b/node_modules/expo-font/ios/EXFont.podspec
@@ -10,16 +10,16 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '12.0'
+  s.platforms     = {:ios => '12.0', :tvos => '12.0'}
   s.source         = { git: 'https://github.com/expo/expo.git' }
   s.static_framework = true
 
   s.dependency 'ExpoModulesCore'
 
-  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
-    s.source_files = "#{s.name}/**/*.h"
-    s.vendored_frameworks = "#{s.name}.xcframework"
-  else
+  # if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
+  #   s.source_files = "#{s.name}/**/*.h"
+  #   s.vendored_frameworks = "#{s.name}.xcframework"
+  # else
     s.source_files = "#{s.name}/**/*.{h,m}"
-  end
+  # end
 end
