diff --git a/node_modules/unimodules-file-system-interface/ios/UMFileSystemInterface.podspec b/node_modules/unimodules-file-system-interface/ios/UMFileSystemInterface.podspec
index 6d128f9..93b0843 100644
--- a/node_modules/unimodules-file-system-interface/ios/UMFileSystemInterface.podspec
+++ b/node_modules/unimodules-file-system-interface/ios/UMFileSystemInterface.podspec
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, '11.0'
+  s.platforms     = {:ios => '11.0', :tvos => '12.0'}
   s.source         = { git: 'https://github.com/expo/expo.git' }
 
   s.source_files = 'UMFileSystemInterface/**/*.{h,m}'
